tests/global_config.py

```diff
@@ -9,6 +9,7 @@
 distribution: str = "downstream"
 applications_namespace: str = "redhat-ods-applications"
 model_registry_namespace: str = "rhoai-model-registries"
+num_mr_instances: int = 2
```

Comment:
I think this shouldn't be in the global config, maybe in our config / constants / utils